.mage/tools.mk

```diff
@@ -1,7 +1,7 @@
 mage_folder := $(abspath $(dir $(lastword $(MAKEFILE_LIST))))
 mage_generated_path := $(mage_folder)/gen
 mage_targets_file := $(mage_generated_path)/targets.mk
-mage := $(mage_generated_path)/local-mage
+mage := $(mage_generated_path)/local-mage 2> /dev/null
```

Comment:
I had to add this because of https://github.com/magefile/mage/pull/399 the `sh.Output` triggers `log.Println` and we call this command a lot. We could wait and hope that the PR gets merged, or we can move forward with this solution